app/javascript/components/Checkout/Receipt.tsx

```diff
@@ -55,11 +56,9 @@ const FailedLineItemResultEntry = ({ name, result }: { name: string; result: Err
         </section>
       </div>
       <div>
-        <div
-          className="payment_failed_notice warning"
-          role="alert"
-          dangerouslySetInnerHTML={{ __html: `<div>${message}</div>` }}
-        />
+        <Alert variant="warning">
+          <div dangerouslySetInnerHTML={{ __html: `<div>${message}</div>` }} />
```

Comment:
```suggestion
          <div dangerouslySetInnerHTML={{ __html: message }} />
```

The extra div seems redundant?